voicevox_engine/synthesis_engine/synthesis_engine.py

```diff
@@ -182,12 +182,22 @@ def speakers(self) -> str:
     def supported_devices(self) -> Optional[str]:
         return self._supported_devices
 
-    def initialize_speaker_synthesis(self, speaker_id: int):
+    def _do_initialize_speaker_synthesis(self, speaker_id: int, skip_reinit: bool):
         try:
             with self.mutex:
-                self.core.load_model(speaker_id)
+                # 以下の条件のいずれかを満たす場合, 初期化を実行する
+                # 1. 引数 skip_reinit が False の場合
+                # 2. 話者が初期化されていない場合
+                if (not skip_reinit) or (not self.core.is_model_loaded(speaker_id)):
+                    print("init!")
```

Comment:
細かいですが、ここのログはもう少し細かい情報を出すか、いっそ省いてしまっても良いかもと思いました！
```suggestion
                    print(f"init! speaker_id:{speaker_id}")
```
```suggestion
```

Reply:
ローカルデバッグ時の消し忘れでした 😿 ありがとうございます！